### DISCUSSION TITLE
How to specify "to_world" directly by a 4x4 matrix

### DISCUSSION THREAD

**Discussion by 07hyx06** [mitsuba-renderer/mitsuba3#670] [Q&A] [answered]
Hi, thanks for the great project and the document. 

I find that to specify the "to_world" parameter in the sensor, some APIs are provided, like look_at, rotate around a specific axis, and translation. If I already have a camera pose matrix (e.g. obtained from COLMAP), can I use this matrix to specify the "to_world" parameter without converting it to axis-angle rotation? 

Thanks.

**Comment by njroussel** [comment, marked as answer]
Hi @07hyx06 

Yes you can, [this constructor](https://mitsuba.readthedocs.io/en/stable/src/api_reference.html#id444) accepts a Matrix as input.

```python
matrix = np.array([
    [0., 1., 2., 3.],
    [1., 2., 3., 4.],
    [2., 3., 4., 5.],
    [3., 4., 5., 6.]
])
transform =  mi.Transform4f(matrix)
```
